src/store/people.js

```diff
@@ -17,17 +29,67 @@ export default {
   actions: {
     async list ({ commit }, page = 1) {
       // Fetch People from SWAPI and await the response.
-      const response = await fetch('https://swapi.co/api/people')
-      // Parse the JSON string returned in the response into a JavaScript
-      // object and destructure the results property out of that object.
-      const { results } = await response.json()
-      if (response.ok) {
-        // If the response is OK commit the data to the store.
-        commit('results', results)
-      } else {
-        // If the response is not OK, log the response to the console.
-        console.error(response)
+      const results = await fetch('https://swapi.co/api/people')
+        .then(function (response) {
+          if (response.ok) {
+            return response.json()
+          }
+        })
+      commit('results', results.results)
```

Comment:
Unfortunately, this change is a step back because it's not handling the error case.